packages/uswds-core/src/styles/settings/_settings-components.scss

```diff
@@ -123,6 +123,11 @@ $theme-in-page-nav-margin-left: 4 !default;
 $theme-in-page-nav-margin-top: 2.5 !default;
 $theme-in-page-nav-top: 4 !default;
 
+// Link
+$theme-external-link-sr-label: "External link" !default;
```

Comment:
No action is required on this, but wondering if we can reduce some audio redundancy by removing the word “link” here so that it is just “External”. In Voiceover, that would mean the readout is “link, _external_, opens in a new tab [link text]”. I am not sure if other screen readers announce “link” in the same order though, so this might just reflect a VoiceOver bias.

Reply:
@amycole501 did you notice this issue on NVDA/JAWS? If so, I can shorten it to just "External".